sys-libs/readline/readline-6.3.8.recipe

```diff
@@ -85,6 +89,7 @@ BUILD()
 INSTALL()
 {
 	make install
+	strip -s $libDir/libhistory.so.6.3 $libDir/libreadline.so.6.3
```

Comment:
it should be strip$secondaryArchSuffix. Also add cmd:strip$secondaryArchSuffix to BUILD_PREREQUIRES.